confidant/authnz/userauth.py

```diff
@@ -586,7 +587,11 @@ def consume_saml_assertion(self):
 
         attributes = auth.get_attributes()
         logger.info('SAML attributes: {!r}'.format(attributes))
-
+        panther_client.send_event({
```

Comment:
Fail open calls to Panther:

https://github.com/lyft/python-lyft-lumos-common/blob/main/lyft_lumos_common/services/panther.py#L42